bin/dash-cli

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -ea
+
+. ./lib/cli/utils.sh
+
+CMD_USAGE="Execute dashd RPC commands
+
+Usage: dash-cli <network_name> <host> <rpc-command> [arguments]"
+
+for i in "$@"
+do
+case ${i} in
+    -h|--help)
+        echo "$CMD_USAGE"
+        exit 0
+    ;;
+esac
+done
+
+NETWORK_NAME="$1"
+HOST_NAME="$2"
+RPC_COMMAND="${@:3}"
+
+. ./lib/cli/init.sh
+. ./lib/cli/ansible.sh
+. ./lib/cli/openvpn.sh
+
+if [[ -z ${HOST_NAME} ]]; then
+    print_error "Host name is required"
+fi
+
+if [[ -z ${RPC_COMMAND} ]]; then
+    print_error "RPC command is required"
+fi
+
+ANSIBLE_HOST=$(ansible_get_ip_by_host ${HOST_NAME})
+
+if [[ -z ${ANSIBLE_HOST} ]]; then
+    print_error "Invalid host name: ${HOST_NAME}. Please use host names from your inventory file '${INVENTORY_FILE}'"
+fi
+
+openvpn_start
+
+NETWORK_ARG=""
+case ${NETWORK} in
+    devnet)
+        NETWORK_ARG="-devnet=${NETWORK_DEVNET_NAME}"
+    ;;
+    testnet)
+        NETWORK_ARG="-testnet"
+    ;;
+    regtest)
+        NETWORK_ARG="-regtest"
+    ;;
+esac
+
+dash-cli -rpcconnect="$ANSIBLE_HOST" \
+         -rpcport=20002 \
```

Comment:
🤔 Are we still using this hard-coded port number everywhere, for all networks?

Reply:
Actually, it is not hardcoded. You can overwrite it for your deployment:

https://github.com/dashpay/dash-network-deploy/blob/master/ansible/group_vars/all#L35
https://github.com/dashpay/dash-network-deploy/blob/master/terraform/aws/variables.tf#L8

Reply:
@nmarley Yep, I was going to read configuration in the next iteration. Probably I need to add comments. 

Actually, I am thinking about rewriting these bash scripts in Go.

Reply:
> Actually, I am thinking about rewriting these bash scripts in Go.

Reply:
> Actually, I am thinking about rewriting these bash scripts in Go.

Hah, I actually thought about that this weekend! 😄  But I'm not certain that it will be worth the effort right now, since these tools work "good enough" for now and are lower priority than other things.